apps/server/lib/lexical/server/code_intelligence/completion/builder.ex

```diff
@@ -84,8 +84,9 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Builder do
       when local_boost in 0..9 and global_boost in 0..9 do
     global_boost = Integer.to_string(9 - global_boost)
     local_boost = Integer.to_string(9 - local_boost)
+    stripped_sort_text = strip_boost(item.sort_text)
```

Comment:
wouldn't this be more clear?
```suggestion
    stripped_sort_text =
     item.label
     |> fallback(item.sort_text)
     |> strip_sort_text()
```

Reply:
Looks good, updated the part